nin/backend/nin

```diff
@@ -15,7 +16,7 @@ program
   .action(function(project){
     project = path.resolve(process.cwd(), project || '');
     console.log('compiling project');
-    compile.compile(project);
+    compile.compile(utils.findProjectRootOrExit(project));
```

Comment:
I'm not quite sure what project was used for, but it looks like you could run a nin project located somewhere else from where you would run the executable.
As relative path would have to point to the root folder of a nin project to work, nothing breaks, as findProjectRoot will find the nin.json located inside the project.

So the requirement is now more lax, you can now type nin run relative/path/to/project/src/whoops instead of just nin run/relative/path/to/project if you would want to do something like that.